docs/kits.rst

```diff
@@ -3,7 +3,27 @@
 CMake Kits
 ##########
 
-**Kits** define the tools used to configure and build a project.
+A *kit* encompasses project-agnostic and configuration-agnostic information
+about how to build code. A kit can include:
+
+- *A set of compilers for some set of languages* - These are locked at specific
+  versions such that you can switch your compiler version quickly and easily.
+- *A Visual Studio installation* - Building for VS is more complicated than
+  simply finding the necessary compiler executable. Visual C++ requires certain
+  environment variables to be set to tell it how to find and link to the
+  Visual C++ toolchain headers and libraries.
+- A toolchain file! This is the lowest-level way to instruct CMake how to
```

Comment:
I think you should stick to surrounding `A toolchain file` with stars `*` and remove the exclamation mark and replace it with `-` just to stay consistent.